app/src/GlobalStyles.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 import { css, Global } from "@emotion/react";
 
-import { ThemeContextType, useTheme } from "./contexts";
+import { useProvider } from "@arizeai/components";
+
+import { ThemeContextType } from "./contexts";
```

Comment:
It may be related to this change but I see the following error in storybook now

<img width="367" alt="image" src="https://github.com/user-attachments/assets/b5cf7b77-b5aa-47c6-adeb-b30c8c910803" />

Reply:
hmm, yeah all the more reason to get all components in I guess. We are loading two anyways so I guess the danger has existed for some time.